lib/pharos/host/el7/scripts/ensure-kubelet.sh

```diff
@@ -6,13 +6,25 @@
 set -e
 
 mkdir -p /etc/systemd/system/kubelet.service.d
-    cat <<EOF >/etc/systemd/system/kubelet.service.d/05-pharos-kubelet.conf
+
+cat <<EOF >/etc/systemd/system/kubelet.service.d/05-pharos-kubelet.conf
 [Service]
 ExecStartPre=-/sbin/swapoff -a
 ExecStart=
 ExecStart=/usr/bin/kubelet ${KUBELET_ARGS} --cgroup-driver=systemd --pod-infra-container-image=${IMAGE_REPO}/pause-${ARCH}:3.1
 EOF
 
+cat <<EOF >/etc/systemd/system/kubelet.service.d/11-cgroups.conf
```

Comment:
Put this here to make it CentOS-only... however, I think we we could also just include it in the `11-pharos.conf` for all hosts... might be clearer. I don't think this should have any side-effects... it should match the default systemd behavior on Ubuntu?